tools/peptideshaker/peptide_shaker.xml

```diff
@@ -7,6 +7,8 @@
     </macros>
     <requirements>
         <requirement type="package" version="1.1">peptide_shaker</requirement>
+        <!-- conda package - both packages should have the same content -->
+        <requirement type="package" version="1.1.3">peptide_shaker</requirement>
```

Comment:
I thought we just said we weren't going to do this. Won't the version 1.1 work as is anyway?